src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(x) {
+      calculator.result += x;
+    },
+
+    subtract(x) {
+      calculator.result -= x;
+    },
+
+    multiply(x) {
+      calculator.result *= x;
+    },
+
+    divide(x) {
+      calculator.result /= x;
+    },
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    operate(operation, x) {
+      operation(x);
+
+      return this;
+    },
+  };
+
+  return calculator;
 }
```

Comment:
Consider using the method shorthand syntax for defining methods within the `calculator` object instead of the `function` keyword.

Reply:
> Consider using the method shorthand syntax for defining methods within the `calculator` object instead of the `function` keyword.

Where am I using function keyword for defining methods?